src/components/Common/Fields/TextField/index.ts

```diff
@@ -0,0 +1 @@
+export { TextField } from './TextField'
```

Comment:
This is high level but I think we're going to want to get away from index.ts for things we want to tree shake away. In this case its probably fine but for the `UI` folder we might not want it to encourage direct imports which can be shaken away.